relay-server/src/envelope.rs

```diff
@@ -150,6 +152,9 @@ impl fmt::Display for ItemType {
             Self::Profile => write!(f, "profile"),
             Self::ReplayEvent => write!(f, "replay_event"),
             Self::ReplayRecording => write!(f, "replay_recording"),
+            Self::CombinedReplayEventAndRecording => {
+                write!(f, "combined_replay_event_and_recording")
```

Comment:
We tend to use short names for everything in our transport protocols, so I would also go for brevity here:

```suggestion
                write!(f, "replay_combined")
```